src/server/pachyderm_test.go

```diff
@@ -2370,12 +2374,12 @@ func TestDeletePipeline(t *testing.T) {
 
 	createPipelines()
 
-	deletePipeline := func(pipeline string) {
-		require.NoError(t, c.DeleteProjectPipeline(pfs.DefaultProjectName, pipeline, false))
+	deletePipeline := func(project, pipeline string) {
+		require.NoError(t, c.DeleteProjectPipeline(project, pipeline, false))
 		time.Sleep(5 * time.Second)
```

Comment:
Why sleep?  Just let the retry logic retry.  (I think this is an effort to make the first attempt succeed, but we should be using the testing backoff, which should just try constantly every 1 second or so.  I don't think backoff.Retry is ever the right thing to use in tests, though.  I recall that you might have fixed the testing version of that, though.)